hoomd/MeshDefinition.cc

```diff
@@ -59,25 +62,31 @@ TriangleData::Snapshot MeshDefinition::getTriangleData()
     }
 
 //! Triangle array setter
-void MeshDefinition::setTriangleData(pybind11::array_t<int> triangles)
+void MeshDefinition::setTriangleData(pybind11::array_t<int> triangles,
```

Comment:
Since you are not checking size this could easily result in a SEGFAULT.

Reply:
I check for size in `mesh.py`